apps/docs/content/docs/sync.mdx

```diff
@@ -54,154 +55,153 @@ function MyApp() {
 
-To use tldraw sync in production, you need to host the backend on your own server. There are three parts to sync in
-tldraw:
+## Self-host the backend
 
-1. The sync server, which serves the room over [WebSockets](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API).
-2. Somewhere to large assets that are too big to send over websockets, like images and videos.
-3. The sync client, which connects to the server with the [`useSync`](?) hook.
+There are two main ways to go about hosting your own backend for tldraw sync:
 
-You can start from our [Cloudflare Workers
-template](https://github.com/tldraw/tldraw-sync-workers). This is a minimal setup of the
-system that powers multiplayer collaboration for hundreds of thousands of rooms & users on
-tldraw.com. It includes everything you need to get up and running with multiplayer tldraw quickly:
-the sync server, asset hosting, and URL unfurling.
+1. Deploy a full backend to Cloudflare using our template (recommended).
+2. Integrate tldraw sync into your own JavaScript backend, using our examples and docs as a guide.
 
-### Connect to your client
+### Use our Cloudflare template
 
-The [`useSync`](?) hook connects your backend to the tldraw editor. Call the hook to create a store,
-then pass it in to the `<Tldraw />` component:
+The best way to get started hosting your own backend is by [cloning and deploying our Cloudflare template](https://github.com/tldraw/cloudlfare-workers-template).
 
-```tsx
-import {useSync} from '@tldraw/sync'
+The template provides a production-grade minimal setup of the system that runs on tldraw.com.
 
-function MyApp() {
-	const store = useSync({
-		uri: 'https://myapp.com/<PATH TO YOUR ROOM>',
-		assets: myAssetStore,
-	})
+It uses:
 
-	return <Tldraw store={store} />
-}
+- [Durable Objects](https://developers.cloudflare.com/durable-objects/) to provide a unique WebSocket server per room.
+- [R2](https://developers.cloudflare.com/r2/) to persist document snapshots and store large binary assets like images and videos.
 
-const myAssetStore: TLAssetStore {
-	upload(file, asset) {
-		return uploadFileAndReturnUrl(file)
-	},
-	resolve(asset) {
-		return asset.props.src
-	},
-}
-```
+There are some features that we have not provided, and you might want to add yourself:
 
-You can see a complete example using the sync client in the
-[`tldraw-sync-cloudflare`](https://github.com/tldraw/tldraw/blob/main/templates/sync-cloudflare/client/App.tsx)
-template.
+- Authentication and authorization.
+- Rate limiting and size limiting for asset uploads.
+- Storing snapshots of documents over time for long-term history.
+- Listing and searching for rooms.
 
-The sync client automatically enables some default collaborative UX which you can customize using
-editor [component overrides](/reference/tldraw/TLComponents):
+Make sure you also read the section below about [deployment concerns](#deployment_concerns).
```

Comment:
Reading through this i sort of got to the end of this section expecting to find out how to actually get going with that template. Could we maybe add a link at the very bottom to the template as a sort of mini cta